lucien-ios/Add New Comic Book/AddComicViewController.swift

```diff
@@ -0,0 +1,224 @@
+        button.semanticContentAttribute = .forceRightToLeft
+        button.imageEdgeInsets.left = button.frame.width - 24
+        button.tintColor = LucienTheme.dark
+    }
+
+    private func configureAddCoverUIButton(button: UIButton) {
+        button.titleLabel?.lineBreakMode = .byWordWrapping
+        button.titleLabel?.textAlignment = .center
+        button.setTitle("Take \n Cover \n Photo", for: .normal)
+        button.titleLabel?.font = LucienTheme.Fonts.Muli.bold.getFont(size: 13)
+        button.setTitleColor(LucienTheme.dark, for: .normal)
+        button.titleEdgeInsets.top = 25
+    }
+
+    private func createPlaceHolderAttributedString(textFieldPlaceholderText: String) -> NSAttributedString {
+        let placeholderAttributes = [
+        NSAttributedStringKey.foregroundColor: LucienTheme.silver,
+        NSAttributedStringKey.font : LucienTheme.Fonts.Muli.regular.getFont(size: 16)
+        ] as [NSAttributedStringKey : Any]
+       return NSAttributedString(string: textFieldPlaceholderText, attributes: placeholderAttributes)
+    }
+
+    @objc private func comicTitleEditingChanged(_ textField: UITextField) {
+        guard let comicTitle = textField.text, !comicTitle.isEmpty else {
+            finishButton.isEnabled = false
+            return
+        }
+        finishButton.isEnabled = true
+        finishButton.tintColor = LucienTheme.dark
+    }
+
+    // MARK: - IBOutlet Methods
+
+    @IBAction func selectADateButtonTapped(_ sender: UIButton) {
+        UIView.animate(withDuration: 0.3, animations: {
+            self.datePicker.isHidden = false
+        })
+    }
+
+    @IBAction func selectGenreButtonTapped(_ sender: UIButton) {
+        UIView.animate(withDuration: 0.3, animations: {
+            self.genrePicker.isHidden = false
+        })
+    }
+
+    @IBAction func selectConditionButtonTapped(_ sender: UIButton) {
+        UIView.animate(withDuration: 0.3, animations: {
+            self.conditionPicker.isHidden = false
+        })
+    }
+
+    // MARK: - UIPickerViewDelegate
+
+    func pickerView(_ pickerView: UIPickerView, numberOfRowsInComponent component: Int) -> Int {
+        return pickerView == genrePicker ? Comic.Genre.allCases.count : Comic.Condition.allCases.count
+    }
+
+    func pickerView(_ pickerView: UIPickerView, titleForRow row: Int, forComponent component: Int) -> String? {
+        return pickerView == genrePicker ? Comic.Genre(rawValue: row)?.title : Comic.Condition(rawValue: row)?.title
+    }
+
+    // MARK: - UIPickerViewDataSource
+
+    func numberOfComponents(in pickerView: UIPickerView) -> Int {
+        return 1
+    }
+
+    // MARK: - UITextFieldDelegate
+
+    func textFieldDidBeginEditing(_ textField: UITextField) {
+        let bottomBorder = textField.layer.sublayers![0] as CALayer
+        bottomBorder.backgroundColor = LucienTheme.dark.cgColor
+
+        guard let comicTitleText = comicTitleTextField.text else {
+            return
```

Comment:
This `return` can be on the same line as the `guard`. You only need to put it on a new line if it's doing something other than returning